CMakeModules/cppcheck.cmake

```diff
@@ -14,6 +14,7 @@ else ()
             --suppress=useStlAlgorithm
             --suppress=noExplicitConstructor
             --suppress=unknownMacro
+            --suppress=unsignedPositive
```

Comment:
why did you need to suppress this one?

Reply:
That was added because of one if in for. I changed it little bit and removed it